swap_meet/vendor.py

```diff
@@ -1,2 +1,91 @@
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory = None):
+        self.inventory = inventory if inventory else []
+
+    def add(self,item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self,item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        return False
```

Comment:
Consider adding a few blank lines within your function to separate logically distinct sections of the code (e.g., setup, processing, return).  According to [PEP 8](https://peps.python.org/pep-0008/#blank-lines), blank lines can improve readability by visually grouping related operations. Right now, everything runs together, which can make it harder to quickly understand the structure and flow of your function.
```suggestion
    def remove(self,item):
        if item in self.inventory:
            self.inventory.remove(item)
            return item
            
        return False
```